llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -13292,12 +13292,15 @@ namespace {
 // apply a combine.
 struct CombineResult;
 
+enum ExtKind : uint8_t { ZExt = 1 << 0, SExt = 1 << 1, FPExt = 1 << 2 };
```

Comment:
Does this need to be a bitmask? I presume an extend can only be one kind at a time

Reply:
It is for refactoring the `canFoldToVWWithSameExtensionImpl`, advised by @wangpc-pp.

Reply:
Ah thanks, I missed that